ivy/src/main/scala/sbt/librarymanagement/ivy/IvyDependencyResolution.scala

```diff
@@ -21,6 +21,19 @@ class IvyDependencyResolution private[sbt] (val ivySbt: IvySbt)
   ): Either[UnresolvedWarning, UpdateReport] =
     IvyActions.updateEither(toModule(module), configuration, uwconfig, log)
 
+  override def withAddedResolvers(
+      resolvers: Seq[Resolver],
+      log: Logger
+  ): IvyDependencyResolution = {
+    val newConfiguration = ivySbt.configuration match {
+      case configuration: ExternalIvyConfiguration =>
+        configuration.withExtraResolvers(configuration.extraResolvers ++ resolvers)
+      case configuration: InlineIvyConfiguration =>
+        configuration.withResolvers(configuration.resolvers ++ resolvers)
```

Comment:
`InlineIvyConfiguration` also has a `otherResolvers` field where the resolvers can be added, not sure what the intended difference is meant to be.